src/main/java/amazonreviewsapp/springboot/controller/ParseController.java

```diff
@@ -0,0 +1,23 @@
+package amazonreviewsapp.springboot.controller;
+
+import amazonreviewsapp.springboot.parser.CSVFileParser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class
+ParseController {
+    private static final String PATH = "src/main/java/amazonreviewsapp/springboot/Reviews.csv";
```

Comment:
it's better to add this file in resources. This way you will be able to get it with only single name.

Reply:
after all let's better remove this constant from controller. Path to the file should not be hardcoded in your application.

Reply:
Sorry, but, unfortunately, I can't get where should I move the file pass, will join you on the QA to clarify that

Reply:
Direct path to the .csv file at the repositories folder added